fuse_core/src/async_publisher.cpp

```diff
@@ -33,71 +33,76 @@
  */
 #include <fuse_core/async_publisher.h>
 
-#include <fuse_core/callback_wrapper.h>
-#include <fuse_core/graph.h>
-#include <fuse_core/transaction.h>
-#include <ros/node_handle.h>
-
-#include <boost/make_shared.hpp>
-
-#include <functional>
-#include <utility>
-#include <string>
+#include <rclcpp/contexts/default_context.hpp>
 
 
 namespace fuse_core
 {
 
 AsyncPublisher::AsyncPublisher(size_t thread_count) :
   name_("uninitialized"),
-  spinner_(thread_count, &callback_queue_)
+  executor_thread_count_(thread_count)
 {
 }
 
 void AsyncPublisher::initialize(const std::string& name)
 {
   // Initialize internal state
   name_ = name;
-  node_handle_.setCallbackQueue(&callback_queue_);
-  private_node_handle_ = ros::NodeHandle("~/" + name_);
-  private_node_handle_.setCallbackQueue(&callback_queue_);
+  std::string node_namespace = "";
+
+  rclcpp::Context::SharedPtr ros_context = rclcpp::contexts::get_global_default_context();
```

Comment:
same comment about TODO to pass in either a context or a node here, pending resolution of https://github.com/locusrobotics/fuse/pull/284/files#r1018501020

Reply:
https://github.com/locusrobotics/fuse/pull/284/commits/7f463a9ba85b274a53f618159fac00ae32b29f05